common/update-page.md

```diff
@@ -8,14 +8,14 @@ button: DPC Google Group
 button_url: "https://groups.google.com/d/forum/dpc-api"
 ---
 
-## Temporary pause on onboarding for DPC pilot
-The Data at the Point of Care (DPC) API supplements patients' health records with data from Fee-For-Service (FFS) Medicare claims. The API is currently in a pilot with an initial group of participants. As an outcome of collaboration and feedback from this pilot group, the DPC team is implementing an improved authorization and onboarding experience. During this process, we've paused onboarding to DPC. **We'll provide an update in July of 2023** with more information on the length of the pause and changes to the onboarding process.
+## Temporary pause on onboarding and applications for DPC pilot
+The Data at the Point of Care (DPC) API supplements patients' health records with data from Fee-For-Service (FFS) Medicare claims. Thanks to collaboration and feedback from our initial pilot group, we're implementing an improved onboarding experience. During this process, **we've paused onboarding and new applications to DPC**. We'll provide an **update** in **July of 2023** on the length of the pause and changes to the onboarding process.
 
 Here's how the temporary pause may affect you:
 
-**If you're interested in DPC, but haven't requested access**
+**If you're interested in DPC, but haven't submitted an application**
 
-Please join us in the [Data at the Point of Care community](https://groups.google.com/d/forum/dpc-api). We'll provide regular updates there, including the date when onboarding will resume.
+We are not taking new applications for production data during the pause. However, you can still request access to the sandbox. We also encourage you to join the [Data at the Point of Care community](https://groups.google.com/d/forum/dpc-api). We'll provide regular updates there, including the date when onboarding will resume.
```

Comment:
There's a link in the confluence page for "request access to the sandbox" — it goes here: https://sandbox.dpc.cms.gov/users/sign_up